### PR TITLE
Replace RGBA static values

### LINKED ISSUE #22360: [EXT] Replace static rgba() values - Part 1

### PR DESCRIPTION
**Pull request by garrettbear**
## **Description**
Replace static RGBA values to a token from `design-tokens` that best matches. If unavailable, a temporary file has been set up for any new tokens needed but not yet provided by the `design-tokens` package. 

## **Related issues**

Fixes: #22360 

## **Manual testing steps**

1. Run locally
2. Compare [local storybook](http://localhost:6006/?path=/docs/getting-started-introduction--docs) to [production storybook](https://metamask.github.io/metamask-storybook/?path=/story/getting-started-introduction--docs) on the components color change
3. Be sure to test light and dark mode

## **Screenshots/Recordings**

See code for before and after photos or video of each change

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [ ] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [x] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ui/pages/send/send.scss [hunk 1]
```diff
@@ -105,7 +105,7 @@
       padding: 16px;
 
       &:hover {
-        background-color: rgba($alto, 0.2);
+        background-color: var(--color-background-hover);
```
[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply ui/pages/send/send.scss:108]: Looks good

Screenshot 2024-01-12 at 2 05 00 PM

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Looking good! Just need to remove the red background color

File: ui/components/app/token-list-display/token-list-display.scss [hunk 2]
```diff
@@ -7,15 +7,15 @@
 }
 
 .token-list-item {
-  padding: 8px;
```
[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply ui/components/app/token-list-display/token-list-display.scss:10]: Ah MB nice catch

**Review by georgewrmarshall — APPROVED** [review]
LGTM! Nice work 🚀

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**